.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+{
+    "tasks": [
+        {
```

Comment:
We either need to fix this so it works and can compile the code or delete it

Reply:
Not sure what you mean

Reply:
I mean this `tasks` section. You can just delete it